### DISCUSSION TITLE
I need re-parse

### DISCUSSION THREAD

**Discussion by KhloyannareK** [CLIUtils/CLI11#812] [Q&A]
Is there a way to clear the parsed results and call parse() again this time for another command line ?
I don't want to create a new parser and add options again.

**Comment by phlptp** [comment]
The `clear()` method will erase all current results.  Though this gets called when calling parse so doing it before won't really have any observable effect.  

What this won't do is reset any variables the parse is setting through the options.  You would have to reset those yourself, or use force_callback on the affected options.  This will always run the option processing either with a given value or a default value and can be used to reset individual flags between multiple parsing.
